phpBB/phpbb/template/twig/tokenparser/event.php

```diff
@@ -26,6 +32,53 @@ class event extends \Twig\TokenParser\AbstractTokenParser
 	public function __construct(\phpbb\template\twig\environment $environment)
 	{
 		$this->environment = $environment;
+		$this->phpbb_dispatcher = $this->environment->get_phpbb_dispatcher();
+
+		$template_event_priority_array = [];
+		/**
+		 * Allow assigning priority to template events
+		 *
+		 * The higher number - the higher tempate event listener priority value is.
+		 * In case of equal priority values, corresponding template event listeners will be handled in default compilation order.
+		 * If not set, template event listener priority will be assigned to the value of 0.
+		 *
+		 * @event core.twig_event_tokenparser_constructor
+		 * @var	array	template_event_priority_array	Array with template event priority assignments per extension namespace
+		 *		Usage:
+		 *		'<author>_<extension_name>' => [
+		 *			'event/<template_event_name>'		=> priority_number,
+		 *		],
+		 *
+		 *		Example:
+		 *		class template_event_order implements EventSubscriberInterface
```

Comment:
Can you create a PR that adds this to the development documentation? I think that would be super helpful for extension developers.

Reply:
Done. https://github.com/phpbb/documentation/pull/307